.github/workflows/build-docs.yaml

```diff
@@ -7,9 +7,11 @@ on:
       - "!main"
     tags-ignore:
       - "*"
+  pull_request:
```

Comment:
I did not specify anything re: branches and tags for this event. I _think_ this is correct, because we won't have any idea what they call it in their fork. 

Also, if there's a better YAML syntax here for one thing that has children (`push`) and one that has no children (`pull_request`), let me know!